config/dev/web-ui-test.json

```diff
@@ -122,9 +122,6 @@
     },
     "infos": {
       "description": true,
-      "dateformat":true,
-      "timeformat":true,
-      "datetimeformat":true,
       "disable": false
```

Comment:
remove disable field
To do as well in web-ui.json and config/docker/web-ui.json